pkg/bpfassets/attacher/attacher_suite_test.go

```diff
@@ -11,3 +14,12 @@ func TestAttacher(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "Attacher Suite")
 }
+
+var _ = Describe("BPF attacher test", func() {
+	It("should attach bpf module", func() {
+		m, err := AttachBPFAssets()
+		// if build with -tags=bcc, the bpf module will be attached successfully
+		Expect(err).NotTo(HaveOccurred())
+		DetachBPFModules(m)
```

Comment:
a function tested without return value?

Reply:
`DetachBPFModules` is to clean up ebpf, it has no return

Reply:
I hope it can return an error if any to ensure no panic. but seems not in this pr.